src/c++/vdo/base/dm-vdo-target.c

```diff
@@ -36,9 +36,11 @@
 #include "logger.h"
 #include "memory-alloc.h"
 #include "message-stats.h"
+#if defined(VDO_INTERNAL) || defined(INTERNAL)
 #ifdef __KERNEL__
 #include "pool-sysfs.h"
```

Comment:
We could combine these #ifdef into one clasue but I suppose it would be a confusing clause.

Do we need more than __KERNEL__ and TEST_INTERNAL here?
 (Also the uds-sysfs.h include farther down).